lcls-twincat-common-components/Library/Library.plcproj

```diff
@@ -15,7 +15,7 @@
     <Implicit_Jitter_Distribution>{4604432a-6eee-489d-ba4c-0a908a6f435f}</Implicit_Jitter_Distribution>
     <LibraryReferences>{78c19189-384c-4a07-a84d-94cfbb123508}</LibraryReferences>
     <CombineIds>false</CombineIds>
-    <Title>lcls-twincat-common-components</Title>
+    <Title>lcls-twincat-cc-km</Title>
```

Comment:
Reminder to switch back to the normal library name before merge

Reply:
Switched back